ngap/handler.go

```diff
@@ -3801,6 +3801,20 @@ func HandleUplinkRanStatusTransfer(ran *context.AmfRan, message *ngapType.NGAPPD
 		return
 	}
 	// send to T-AMF using N1N2MessageTransfer (R16)
+
+	// Send Downlink RAN Status Transfer
+	targetUe := ranUe.TargetUe
+	if targetUe != nil {
+		logger.NgapLog.Info("Target AMF UE NGAP Id: ", targetUe.AmfUeNgapId)
+		logger.NgapLog.Info("Target RAN UE NGAP Id: ", targetUe.RanUeNgapId)
+	}
+
+	if rANStatusTransferTransparentContainer != nil {
+		ngap_message.SendDownlinkRanStatusTransfer(amfUe.RanUe[models.AccessType__3_GPP_ACCESS], *rANStatusTransferTransparentContainer)
+		ngap_message.SendDownlinkRanStatusTransfer(targetUe, *rANStatusTransferTransparentContainer)
```

Comment:
What if `targetUe` is `nil`? Are not you checking that because it is getting checked in `SendDownlinkRanStatusTransfer()`? or why are not you checking it?

Reply:
Yes, I didn't add a nil check because the SendDownlinkRanStatusTransfer function already handles it for ue (i.e., targetUe). However, I can add an additional nil check if we want to log an early message or perform other actions when targetUe is nil. Let me know if you think logging or any specific action should be done in that case.

Reply:
No need to add another check. Thr main thing was that I wanted to double check I understood the reason for that.